src/components/encoding-pane/encoding-shelf.tsx

```diff
@@ -18,6 +19,7 @@ import {DraggedFieldIdentifier, Field} from '../field/index';
 import * as styles from './encoding-shelf.scss';
 import {FieldCustomizer} from './field-customizer';
 import {FunctionPicker, FunctionPickerWildcardHandler} from './function-picker';
+import { CUSTOMIZABLE_ENCODING_CHANNELS } from './property-editor-schema';
```

Comment:
Please do not include space between braces.  If we don't have this a part of linter yet, please submit a separate PR to enforce this.

Reply:
Sorry every time I use VS Code this happens. Will make a PR to add this rule to linter.